.github/workflows/deploy-tag.yml

```diff
@@ -1,53 +0,0 @@
-name: Deployment
-
-on:
-  push:
-    tags:
-      - "20.*"
-      - "21.*"
-      - "22.*"
-      - "23.*"
-      - "24.*"
-
-jobs:
-
-  build:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - run: npm ci
-    - run: npm run build
-    - name: Update extension’s meta
-      run: |
-        export VER=$(echo "$GITHUB_REF" | sed -e 's/refs\/tags\///')
-        echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > dist/SOURCE_URL
-        npm run version -- $VER
-    - uses: actions/upload-artifact@40fe781 # TODO: update when v2 leaves beta
-      with:
-        path: dist
-
-  deploy-to-chrome:
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/download-artifact@v1
-      with:
-        name: artifact
-    - run: cd artifact && npx chrome-webstore-upload-cli@1 upload --auto-publish
-      env:
-        EXTENSION_ID: kgilejfahkjidpaclkepbdoeioeohfmj
```

Comment:
@tanmayrajani Can you set this as a repository secret? I don't have access to this repo’s settings

Reply:
Ping

Reply:
If you'd like to transfer the repo/admin to me I can add them myself

Reply:
I'm really sorry, I'm not sure why I'm seeing this for the first time :neutral_face: 
I just added `EXTENSION_ID` in the secrets